### ISSUE TITLE
Problem with callingWaterfall

### ISSUE THREAD

**Issue by ron1891** [bhklab/CoreGx#120]
```
> callingWaterfall(
+   x= t_AUC,
+   type = "AUC",
+   intermediate.fold = 1.2,
+   cor.min.linear = 0.95,
+   name = "Drug",
+   plot = FALSE
+ )
**Error in .distancePointLine(x = x[1], y = x[2], a = slope, b = intercept)  argument "c" is missing, with no default.**
```

Hi, I have been trying to run callingWaterfall function but an error related to _.distancePointLine_ which needs 'c' as an input, but when I looked into the raw code, .distancePointLine did not have 'c' as its input. Here's the raw code regarding this issue.
![image](https://user-images.githubusercontent.com/89532424/130813325-1d33b3b5-e5dd-4443-9a7c-f9f8ff68c5ff.png)
However, when we check on the .distancePointLine function it needed five arguments, which included 'c'.
![image](https://user-images.githubusercontent.com/89532424/130813734-75ebc781-6a66-4a8c-a6a0-23851e595baa.png)

I would like to know if there's something I have been missing regarding this code.

Thanks.

Han

**Comment by ChristopherEeles** [comment]
Hi @ron1891,

I just pushed a fix for this to the Bioconductor 3.13 release. It should be available in Bioconductor in a few days, assuming it passes the automated checks. Look for version 1.4.2.

If you are in a hurry, you could also install it from GitHub with:
```
remotes::install_github("bhklab/CoreGx@RELEASE_3_13")
```

If you are using Bioconductor 3.14 then you will have to wait until I update `master` and push it to Bioconductor.

I have a PR open right now and will close this issue once it gets merged.

Best,
Christopher Eeles | Software Developer | BHK Lab @ UHN

**Comment by ChristopherEeles** [comment]
Hi @ron1891,

FYI, the fix is live on the RELEASE (3.13) branch of Bioconductor as of today. I am still working on the PR to push changes to the DEVELOPMENT (3.14) branch.

Best,
Chris

**Comment by ChristopherEeles** [comment]
Fix on DEVELOPMENT (3.14) branch of Bioconductor has been pushed. Look for version 5.15 of CoreGx.
